apps/ui/src/views/Space/Proposals.vue

```diff
@@ -83,7 +115,10 @@ watchEffect(() => setTitle(`Proposals - ${props.space.name}`));
 
 <template>
   <div>
-    <div class="flex justify-between p-4 gap-2">
+    <div
+      class="flex justify-between p-4 gap-2 gap-y-3 flex-row"
+      :class="{ 'flex-col-reverse sm:flex-row': space.labels?.length }"
+    >
```

Comment:
Not sure if we want to use different layouts with/without labels. @bonustrack could you check it in mobile?

Reply:
In my mind, every mobile layout will use the 2 lines version, once we're adding more filter options, like https://github.com/snapshot-labs/sx-monorepo/issues/460